config/initializers/datadog_apm.rb

```diff
@@ -6,6 +6,7 @@
   c.tracer enabled: Rails.env.production?
   c.tracer partial_flush: true
   c.tracer priority_sampling: true
+  c.use :elasticsearch
```

Comment:
This APM setting does not appear related to the changes in this PR unless it's me missing something which is highly possible ;)

Reply:
GDI! 🤦‍♀